### ISSUE TITLE
No error for unsupported media files

### ISSUE THREAD

**Issue by pritdeveloper** [davinder17s/unplay#1]
There should be an error for the unsupported media files.
maybe [this repo](https://github.com/CodeSeven/toastr) will be useful.

**Comment by davinder17s** [comment]
Yup thanks for review bro :-) hale just beta hai, bht kuj plan kita me agge
v islyi :-D, hope ke kafi useful project bnega eh, agr nodejs da package
bna dyie ta fr local files ape scan krke libraries bna lu etc etc
